src/tools/crossgen/crossgen.cpp

```diff
@@ -422,6 +422,7 @@ int _cdecl wmain(int argc, __in_ecount(argc) WCHAR **argv)
     LPWSTR pwzSearchPathForManagedPDB = NULL;
     LPCWSTR pwzOutputFilename = NULL;
     LPCWSTR pwzPublicKeys = nullptr;
+    bool fExplicitReadyToRunSwitch = true;
```

Comment:
Shouldn't this actually be default to false?

Reply:
thanks for catching this